diff --git a/node_modules/react-dom/cjs/react-dom-client.development.js b/node_modules/react-dom/cjs/react-dom-client.development.js
index 0cc753e..0c9af46 100644
--- a/node_modules/react-dom/cjs/react-dom-client.development.js
+++ b/node_modules/react-dom/cjs/react-dom-client.development.js
@@ -30886,15 +30886,15 @@
         -1 === navigator.userAgent.indexOf("Edge")) ||
         -1 < navigator.userAgent.indexOf("Firefox"))
     ) {
-      var protocol = window.location.protocol;
-      /^(https?|file):$/.test(protocol) &&
-        console.info(
-          "%cDownload the React DevTools for a better development experience: https://react.dev/link/react-devtools" +
-            ("file:" === protocol
-              ? "\nYou might need to use a local HTTP server (instead of file://): https://react.dev/link/react-devtools-faq"
-              : ""),
-          "font-weight:bold"
-        );
+      // var protocol = window.location.protocol;
+      // /^(https?|file):$/.test(protocol) &&
+      //   console.info(
+      //     "%cDownload the React DevTools for a better development experience: https://react.dev/link/react-devtools" +
+      //       ("file:" === protocol
+      //         ? "\nYou might need to use a local HTTP server (instead of file://): https://react.dev/link/react-devtools-faq"
+      //         : ""),
+      //     "font-weight:bold"
+      //   );
     }
     exports.createRoot = function (container, options) {
       if (!isValidContainer(container))
